PSKoans/Data/Show-MeditationPrompt.Data.psd1

```diff
@@ -192,6 +192,39 @@ of your own catastrophe.
         @'
 When you realise how perfect everything is,
 you will tilt your head back and laugh at the sky.
+'@
+        @'
+If you face a problem,
+it is not the problem that will detour.
+'@
+        @'
+You only lose what you cling to.
+'@
+        @'
+Do not dwell in the past,
+do not dream of the future,
+concentrate the mind on the present moment.
+'@
+        @'
+Never fear the unknown,
+thus the unknown does not fear you.
+'@
+        @'
+Possibilities are here, there, and wverywhere.
+Keep your mind open to even more.
+'@
+        @'
+The most life gives us,
+often comes from failure at first.
+'@
+        @'
+When it gets tough,
+keep believeing that
+everyhting will change for the better.
```

Comment:
```suggestion
everything will change for the better.
```

Reply:
okay, now I start to feel embarassed about the amount of typos...